standard/standard-library.md

```diff
@@ -658,3 +652,400 @@ namespace System.Runtime.CompilerServices
     }
 }
 ```
+
+## C.4 Format Specifications
+
+The meaning of the formats, as used in interpolated string expressions (§interpolated-string-expressions), are defined in ISO/IEC 23271:2012. For convenience the following text is copied from the description of `System.IFormatable`.
+
+**This text is informative.**
+
+A *format* is a string that describes the appearance of an object when
+it is converted to a string. Either standard or custom formats can be used. A
+standard format takes the form *Axx*, where *A* is a single
+alphabetic character called the *format specifier*
+, and *xx* is an integer between zero and 99 inclusive, called the *precision specifier*. The format specifier controls the type
+of formatting applied to the value being represented as a string. The
+*precision specifier* controls the number 
+of significant digits or decimal places in the string, if applicable. [*Note:* For the list of standard format specifiers, see the
+table below. Note that a given data type, such as `System.Int32`
+, might not support one
+or more of the standard format specifiers.]
+
+[*Note:* When a format includes symbols that vary by culture, such as the currency
+symbol included by the ‘C’ and ‘c’ formats, a formatting object supplies the
+actual characters used in the string representation. A method might include a
+parameter to pass a `System.IFormatProvider` object that supplies a
+formatting object, or the method might use the default formatting object, which
+contains the symbol definitions for the current culture. The current culture
+typically uses the same set of symbols used system-wide by default. In the Base
+Class Library, the formatting object for system-supplied numeric types is a
+`System.Globalization.NumberFormatInfo` instance. For `System.DateTime` instances, a 
+`System.Globalization.DateTimeFormatInfo` is
+used.]
+
+The following table describes the standard format specifiers and associated formatting
+object members that are used with numeric data types in the Base Class
+Library.
+
+<table>
```

Comment:
I seem to remember that I make very specific assumptions about the formats of tables in the Word converter. This may very well not work at the moment.

Reply:
I think it looks like the other tables, and the Word Convertor didn't barf but I've not seen its output…

Reply:
Jon to attach a screenshot after fetching and running the Word converter.

Reply:
Nope, the Word converter fails entirely for this. This will take a fair amount of work... I'll see if the same table converter that manages to handle the table in the unsafe types section would work here - but it does mean a rather larger tweak of the XML we're "copying" the content from.